cmd/benchcheck/benchcheck.go

```diff
@@ -66,7 +66,20 @@ func main() {
 		log.Fatal("-new is obligatory")
 	}
 
-	results, err := benchcheck.StatModule(*mod, *oldRev, *newRev)
+	runBench := benchcheck.DefaultRunBench
+	var customizedBenchCmd []string
+
+	for i, v := range os.Args {
```

Comment:
you can use [flag.Args()](https://pkg.go.dev/flag#Args) to only iterate in the non-flag remaining args.